{{cookiecutter.project_slug}}/frontend/.devcontainer/docker-compose.extend.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
```

Comment:
copy pasted from react app

Reply:
You can remove this, we don't use it with Tilt and k8s.